docs/octopus-rest-api/octopus.client.md

```diff
@@ -99,37 +113,48 @@ The repository methods all make direct HTTP requests, there's no "session" abstr
 
 For some operations not available through repositories it will be necessary to use the `IOctopusClient` type:
 
-```cs
+```powershell PowerShell
+$connection = $repository.Client.Get($machine.Links["Connection"]);
+```
+```cs C#
 // Sync
 var connection = repository.Client.Get(machine.Links["Connection"]);
  
 // Async
 var connection = await client.Get(machine.Links["Connection"]);
 ```
 
-The entire API is accessible by traversing links - each resource carries a collection of links, like the "Connection" link on `MachineResource` shown above.
+The entire API is accessible by traversing links - each resource carries a collection of links, like the `Connection` link on `MachineResource` shown above.
+
+:::warning
+Always access objects by traversing the links - avoid using direct url segments, as they may change in the future.
```

Comment:
I think a semi colon is the correct punctuation here, unless the dash could be replaced with the word 'and', but I think the intent is for readers to follow the instruction in the first clause, with the second clause 'avoid using...' intended as an additional reason to follow the first instruction.
```suggestion
Always access objects by traversing the links; avoid using direct url segments, as they may change in the future.
```